src/OrbitGl/TimeGraph.h

```diff
@@ -69,7 +70,7 @@ class TimeGraph : public orbit_gl::CaptureViewElement {
   [[nodiscard]] const orbit_client_data::CaptureData* GetCaptureData() const {
     return capture_data_;
   }
-  [[nodiscard]] TrackManager* GetTrackManager() { return track_manager_.get(); }
+  [[nodiscard]] TrackManager* GetTrackManager() const { return track_manager_.get(); }
```

Comment:
Since you made that function `const`, was it also your intention to return a `const TrackManager*`?

Reply:
yes, why not.

Reply:
There are a few classes using that function to make some non-const changes.

Should be straightforward after the split between GetTrack methods and Create ones

Reply:
ok, if TrackManager returned here is not const the function should also not be const because this class owns the TrackManager.

Reply:
I think you forgot to remove const here?

Reply:
Didn't push the commit yet, thanks